README.md

```diff
@@ -142,17 +144,18 @@ Lets take a look at sample `FCM` service configuration:
 ```elixir
 config :mongoose_push, fcm: [
     default: [
-        key: "fake_app_key",
+        tokenfile: "path/to/token.json",
```

Comment:
`appfile`? This file does not contain any tokens, only X509 private key. And since this is not *only* key, I think more generic could be used.